bears/js/JSStandardBear.py

```diff
@@ -24,5 +25,17 @@ class JSStandardBear:
     CAN_FIX = {'Formatting'}
     SEE_MORE = 'https://standardjs.com/rules.html'
 
-    def create_arguments(self, filename, file, config_file):
-        return (filename, '--verbose')
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         parser: str = '',
+                         globals: typed_list(str)= (),
+                         plugin: str = ''):
```

Comment:
The idea is actually to implement the rules of `standard` directly, the user shouldn't deal with plugins and stuff, but just say `lint-xyz = True`.

Reply:
so as I understand here `lint-xyz` would be the name of a parser? @Makman2

Reply:
no the capability of a parser, like `remove_unnecessary_spaces` or `disallow_strings` so (if that would exist).

Reply:
The idea of coala is to abstract those things like "parsers", "plugins" etc etc away. The user has our bears, which accept settings that directly correlate to a certain analyis. The user shall not deal with internals of tools like `standard`, i.e. that it uses plugins and parsers and globals.

Reply:
so i need to add settings for the user according to the capabilities of the different parsers that can be used?  @Makman2

Reply:
exactly ;)